options/locale/locale_en-US.ini

```diff
@@ -2225,6 +2225,10 @@ teams.leave = Leave
 teams.leave.detail = Leave %s?
 teams.can_create_org_repo = Create repositories
 teams.can_create_org_repo_helper = Members can create new repositories in organization. Creator will get administrator access to the new repository.
+teams.none_access = None Access
+teams.none_access_helper = Members can view or do any other action on this unit.
+teams.general_access = General Access
+teams.general_access_helper = Members permissions will be decided by below permission table.
 teams.read_access = Read Access
 teams.read_access_helper = Members can view and clone team repositories.
```

Comment:
Looks like this `team.read_access_helper` message is no longer used? Same for `teams.write_access_helper`.

Reply:
I added tooltip on the column heads